env0/resource_project_policy_test.go

```diff
@@ -48,110 +49,107 @@ func TestUnitPolicyResource(t *testing.T) {
-					resource.TestCheckResourceAttr(accessor, "include_cost_estimation", strconv.FormatBool(policy.IncludeCostEstimation)),
-					resource.TestCheckResourceAttr(accessor, "skip_apply_when_plan_is_empty", strconv.FormatBool(policy.SkipApplyWhenPlanIsEmpty)),
-					resource.TestCheckResourceAttr(accessor, "disable_destroy_environments", strconv.FormatBool(policy.DisableDestroyEnvironments)),
-					resource.TestCheckResourceAttr(accessor, "skip_redundant_deployments", strconv.FormatBool(policy.SkipRedundantDeployments)),
-					resource.TestCheckResourceAttr(accessor, "run_pull_request_plan_default", strconv.FormatBool(policy.RunPullRequestPlanDefault)),
-					resource.TestCheckResourceAttr(accessor, "continuous_deployment_default", strconv.FormatBool(policy.ContinuousDeploymentDefault)),
-					resource.TestCheckResourceAttr(accessor, "max_ttl", "inherit"),
-					resource.TestCheckResourceAttr(accessor, "default_ttl", "inherit"),
-				),
-			},
-			{
-				Config: resourceConfigCreate(resourceType, resourceName, map[string]interface{}{
-					"project_id":                    updatedPolicy.ProjectId,
-					"number_of_environments":        "null",
-					"number_of_environments_total":  "null",
-					"requires_approval_default":     updatedPolicy.RequiresApprovalDefault,
-					"include_cost_estimation":       updatedPolicy.IncludeCostEstimation,
-					"skip_apply_when_plan_is_empty": updatedPolicy.SkipApplyWhenPlanIsEmpty,
-					"disable_destroy_environments":  updatedPolicy.DisableDestroyEnvironments,
-					"skip_redundant_deployments":    updatedPolicy.SkipRedundantDeployments,
-					"max_ttl":                       "Infinite",
-					"default_ttl":                   *updatedPolicy.DefaultTtl,
-				}),
-				Check: resource.ComposeAggregateTestCheckFunc(
-					resource.TestCheckResourceAttr(accessor, "project_id", updatedPolicy.ProjectId),
-					resource.TestCheckNoResourceAttr(accessor, "number_of_environments"),
-					resource.TestCheckNoResourceAttr(accessor, "number_of_environments_total"),
-					resource.TestCheckResourceAttr(accessor, "requires_approval_default", strconv.FormatBool(updatedPolicy.RequiresApprovalDefault)),
-					resource.TestCheckResourceAttr(accessor, "include_cost_estimation", strconv.FormatBool(updatedPolicy.IncludeCostEstimation)),
-					resource.TestCheckResourceAttr(accessor, "skip_apply_when_plan_is_empty", strconv.FormatBool(updatedPolicy.SkipApplyWhenPlanIsEmpty)),
-					resource.TestCheckResourceAttr(accessor, "disable_destroy_environments", strconv.FormatBool(updatedPolicy.DisableDestroyEnvironments)),
-					resource.TestCheckResourceAttr(accessor, "skip_redundant_deployments", strconv.FormatBool(updatedPolicy.SkipRedundantDeployments)),
-					resource.TestCheckResourceAttr(accessor, "max_ttl", "Infinite"),
-					resource.TestCheckResourceAttr(accessor, "default_ttl", *updatedPolicy.DefaultTtl),
-				),
+	t.Run("create and update", func(t *testing.T) {
```

Comment:
Not too many changes. 
1. It's now under a subtest.
2. Fixed the place where is caused a crash (get a value for nil pointer --- ooops).